components/grid/grid.css

```diff
@@ -0,0 +1,88 @@
+*{
+    --grid-border-color:dodgerblue;
+}
+
+.two-column-grid-container{
+    display:grid;
+    grid-template-areas: 'one two'
+                        'three four';
+                        grid-gap:10px;
+   
+}
+.two-column-grid-container .one,.two-column-grid-container .two,.two-column-grid-container .three , .two-column-grid-container .four{
+    padding:10px;
+    text-align: center;
+    
+    border: 1px solid var(--grid-border-color);
+   
+}
+
+.three-column-grid-container{
+    display: grid;
+    grid-template-areas: "one two three"
+                         "four five six"
+                         "seven eight nine";
+                         grid-gap:10px;
+}
+.three-column-grid-container .one,.three-column-grid-container .two,.three-column-grid-container .three,.three-column-grid-container .four,.three-column-grid-container .five,.three-column-grid-container .six,.three-column-grid-container .seven, .three-column-grid-container .eight ,.three-column-grid-container .nine{
+    padding:10px;
+    text-align: center;
+    border:1px solid var(--grid-border-color);
+    
+}
+
+
+.common-layout-grid-container >*{
+    display: flex;
+    border: 1px solid var(--grid-border-color);
+    justify-content: center;
+    text-align: center;
+    align-items: center;
```

Comment:
u can make a utility class called flex-center for properties { display: flex , align-items: center and justify-content: center } since we frequently center things